Simplenote/Resources/AppStoreStrings.pot

```diff
@@ -66,9 +66,8 @@ msgctxt "app_store_keywords"
 msgid "simplenote,note,taking,taker,notes,sync,share,simple,markdown,tag,to-do,list,lock,cloud,writing"
 msgstr ""
 
-msgctxt "v4.22.0-whats-new"
+msgctxt "v4.23-whats-new"
```

Comment:
This is inline with the `glotPressWhatsNewKey` and is back to the usual `major.minor` format. After the change from #833.